core/extension/gdextension_interface.cpp

```diff
@@ -1077,10 +1077,12 @@ static GDExtensionMethodBindPtr gdextension_classdb_get_method_bind(GDExtensionC
 		return nullptr;
 	}
 	ERR_FAIL_COND_V(!mb, nullptr);
+#ifndef LIBRARY_ENABLED
 	if (mb->get_hash() != p_hash) {
 		ERR_PRINT("Hash mismatch for method '" + classname + "." + methodname + "'.");
 		return nullptr;
 	}
+#endif
```

Comment:
Why do we want to disable checking for hash mismatches here? Given that GDExtension does ptrcalls to any methods returned, calling a method with an unexpected signature can lead to crashes.